test/integration_test/kubevirt_test.go

```diff
@@ -246,3 +216,87 @@ func createDatadiskTemplates(t *testing.T) error {
 	}
 	return nil
 }
+
+func kubevirtVMScaledDeployAndValidate(t *testing.T, instanceID string, appKeys []string, scale int) []*scheduler.Context {
+	var allCtxs []*scheduler.Context
+	logrus.Infof("Deploying %d VMs and validating them.", scale)
```

Comment:
nit: this is misleading since appKeys could have multiple apps. So, the number of VMs is `len(appKeys) * scale`